identity/src/keypair.rs

```diff
@@ -152,8 +149,7 @@ impl Keypair {
 
     #[cfg(feature = "ecdsa")]
     #[deprecated(
-        since = "0.2.0",
-        note = "This method name does not follow Rust naming conventions, use `Keypair::try_into_ecdsa` instead."
+        note = "This method name does not follow Rust naming conventions, use `Keypair::from` instead."
```

Comment:
This one needs to be adjusted too please.

Reply:
Fixed